test-suite/src/test/scala/org/scalajs/testsuite/compiler/ReflectionTest.scala

```diff
@@ -76,4 +114,10 @@ object ReflectionTest extends JasmineTest {
 
   class RenamedTestClass
 
+  @JSName("ReflectionTestRawJSClass")
+  class ReflectionTestRawJSClass extends js.Object
+
+  trait ReflectionTestRawJSTrait extends js.Object
+  object ReflectionTestRawJSObject extends js.Object
```

Comment:
Ah yes, but then I realized that there's no way to get a hand on the `j.l.Class` representing that object, so I couldn't write the test. I forgot to remove the definition of the object.